config/routes.rb

```diff
@@ -72,18 +72,18 @@
       get :nyckelpiga, on: :collection
     end
 
-    # A scope to put car-associated things under /bil
-    scope :bil do
-      namespace :admin do
-        resources :rents, path: :bokningar, except: [:index, :edit] do
-          get :preview, path: :visa, on: :member
-        end
-        get '', controller: :rents, action: :main, as: :car
+    namespace :admin do
+      resources :rents, path: :bilbokning, except: :edit do
+        get :preview, path: :visa, on: :member
       end
-      resources :rents, path: :bokningar do
-        patch :authorize, on: :member, path: :auktorisera
+    end
+
+    resources :rents, path: :bilbokning, except: :index do
+      collection do
+        get :oversikt, action: :index, as: :overview
+        get "", action: :main, as: :main
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.